### ISSUE TITLE
Are there any restrictions on the values of alpha? Eg. the sum of alphas = 1

### ISSUE THREAD

**Issue by Saadmaghani** [AdeelH/pytorch-multi-class-focal-loss#4]

**Comment by AdeelH** [comment]
Nope. `alpha` is passed to `nn.NLLLoss` as the `weight` param (https://github.com/AdeelH/pytorch-multi-class-focal-loss/blob/f15f44014542de6927b38b518e36083dfed45911/focal_loss.py#L48-L49) and its [docs](https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html#torch.nn.NLLLoss) don't mention anything about this either.
